server/cluster/metadata/cluster_metadata.go

```diff
@@ -234,7 +234,7 @@ func (c *ClusterMetadata) CreateTableMetadata(ctx context.Context, request Creat
 	}
 
 	if exists {
-		return CreateTableMetadataResult{}, ErrTableAlreadyExists
+		return CreateTableMetadataResult{}, errors.WithMessagef(ErrTableAlreadyExists, "table name %s", request.TableName)
```

Comment:
Shall we follow the style of other logs:
```suggestion
		return CreateTableMetadataResult{}, errors.WithMessagef(ErrTableAlreadyExists, "tableName:%s", request.TableName)
```